src/urllib3/connectionpool.py

```diff
@@ -461,7 +473,16 @@ def _make_request(
 
         # Receive the response from the server
         try:
-            httplib_response = conn.getresponse()
+            response = conn.getresponse(
+                url,
+                method,
+                self,
+                retries,
+                preload_content,
+                decode_content,
+                response_conn,
+                enforce_content_length,
+            )
```

Comment:
pretty large breaking change in the API. I think I want to move `response_conn` up before `self` and maybe do `conn=response_conn` so it's clear for the next person what is going on.